### ISSUE TITLE
Pronk doesn't buid

### ISSUE THREAD

**Issue by sfermigier** [bos/pronk#8]
There is no hint in the README that says how to build it, so I guess I should try:

```
$ cabal update
$ cabal install
```

However, it fails with:

cabal: Error: some packages failed to install:
aeson-0.3.2.12 depends on template-haskell-2.6.0.0 which failed to install.
criterion-0.5.1.0 depends on template-haskell-2.6.0.0 which failed to install.
http-enumerator-0.7.1.3 depends on monad-control-0.2.0.3 which failed to
install.
monad-control-0.2.0.3 failed during the configure step. The exception was:
user error (The package requires Cabal library version -any && >=1.9.2 but no
suitable version is installed.)
pronk-0.1.0.0 depends on monad-control-0.2.0.3 which failed to install.
statistics-0.9.0.0 depends on template-haskell-2.6.0.0 which failed to
install.
template-haskell-2.6.0.0 failed during the building phase. The exception was:
ExitFailure 1

**Comment by sfermigier** [comment]
Mmm, cabal was not up to date. It's still failing, though:

```
Language/Haskell/TH/PprLib.hs:55:0:
    Illegal instance declaration for `Show Doc'
        (All instance types must be of the form (T t1 ... tn)
         where T is not a synonym.
         Use -XTypeSynonymInstances if you want to disable this.)
    In the instance declaration for `Show Doc'
cabal: Error: some packages failed to install:
aeson-0.3.2.12 depends on template-haskell-2.6.0.0 which failed to install.
criterion-0.5.1.0 depends on template-haskell-2.6.0.0 which failed to install.
pronk-0.1.0.0 depends on template-haskell-2.6.0.0 which failed to install.
statistics-0.9.0.0 depends on template-haskell-2.6.0.0 which failed to
install.
template-haskell-2.6.0.0 failed during the building phase. The exception was:
ExitFailure 1
```

**Comment by paulcavallaro** [comment]
Largely useless, but I solved the same problem by merely doing the following:

```
ghc-pkg unregister cmdargs
ghc-pkg unregister template-haskell
cabal install
```

I think I had some sort of weird mismatched dependencies.

**Comment by hvr** [comment]
this may have been fixed by the pull-request attached to issue #13

**Comment by bos** [comment]
Should be fixed as of 0fab265749585c7283cb6865568e3d6422e2a1b2, thanks.
